server/lib/routes/goals.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const client = require('../db-client');
+const Router = express.Router;
+const router = Router(); //eslint-disable-line new-cap
+
+router 
+  .get('/', (req, res) => {
+    client.query(`
+    SELECT id, title, start_date, end_date 
```

Comment:
```suggestion
    SELECT id, title, start_date, to_char(end_date, 'mm/dd/yyyy') as end_date
```
Alternative date formatting on back-end (e.g. returns 12/31/2018).